src/features/favorites/atoms/Buttons/ReinitializeFilters.tsx

```diff
@@ -0,0 +1,25 @@
+import { t } from '@lingui/macro'
+import React from 'react'
+import { TouchableOpacity } from 'react-native'
+
+import { useFavoritesState } from 'features/favorites/pages/FavoritesWrapper'
+import { _ } from 'libs/i18n'
+import { ColorsEnum, Typo } from 'ui/theme'
+import { ACTIVE_OPACITY } from 'ui/theme/colors'
+
+export const ReinitializeFilters = () => {
+  const { dispatch, filter } = useFavoritesState()
+
+  function reinitializeFilters() {
```

Comment:
Ca simplifie la vie de tout le monde :
```suggestion
  function resetFilters() {
```

Reply:
Je vais supprimer ce composant (voir commentaire https://passculture.atlassian.net/browse/PC-7722)

Reply:
ok